CHANGELOG.md

```diff
@@ -32,6 +32,15 @@ _Code:_
 
   - Minor dependency updates making the world a better place ([#680](https://github.com/cossacklabs/themis/pull/680)).
 
+- **Swift**
+
+  - Updated Swift examples (iOS and macOS, Carthage and CocoaPods) to showcase usage newest SecureCell API: generating symmetric keys and using Secure Cell with Passphrase ([#688](https://github.com/cossacklabs/themis/pull/688)).
+
+- **ObjC**
```

Comment:
```suggestion
- **Objective-C**
```

Previous changelog entries grouped both Objective-C and Swift into a single group “iOS and macOS”. However, I think it's a good idea to keep them separated, since these are _language-level_ changes.